styles.css

```diff
@@ -831,3 +824,59 @@ ul {
     margin-bottom: 3%;
   }
 }
+
+@media (max-width: 768px) {
+  .hamburger {
+    display: block;
+    cursor: pointer;
+    margin-left: 86%;
+  }
+
+  .hamburger .line {
+    width: 25px;
+    height: 3px;
+    background-color: #67798e;
+    margin: 6px 0;
+    margin-right: 2%;
+  }
+
+  .menu-section {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    gap: 5%;
+    left: 0;
+    top: 60px;
+    width: 100%;
+    background-color: #fff;
+    transition: 0.5s;
+    height: 450px;
+    overflow: hidden;
+    visibility: visible;
+    opacity: 0;
+  }
```

Comment:
- [ ] Please kindly consider adding the blur effect. In addition the mobile menu has border radius on bottom. This is to ensure that we stick to the Figma template design and also have a professional portifolio ☺️.

Reply:
Hi @AlphaNtihinduka
what should I add like a blur effect?